nx-X11/programs/Xserver/hw/nxagent/Args.c

```diff
@@ -1483,6 +1483,25 @@ static void nxagentParseOptions(char *name, char *value)
 
     return;
   }
+  else if (!strcmp(name, "keyconv"))
```

Comment:
So we don't want to support this as an command line argument as well?

Don't have a strong opinion on this, just asking.

Reply:
It partly depends on the client side if keycode conversion is useful or not. If we make this a command line option we cannot change the setting afterwards. And it is really annoying having a session that is reporting wrong keycodes and you can't fix that easily (I ended up in such a situation recently having to undo the keycode conversion via xmodmap...)

Reply:
Good point. Then again, if the same option is also specified in an options file or string, then it is possible to override this at run time, since the options file or string is re-parsed and applied upon reconnects, overriding the value passed at start time.

I can follow our argumentation, though. It's not really necessary.